src/main/java/com/assignment/Assignment2.java

```diff
@@ -0,0 +1,21 @@
+package com.assignment;
+
+public class Assignment2 {
+    public static String stringTest1() {
+        // Replace Ram to Sam
+        String r = "Ram".replace('R', 'S');
+        return r;
+    }
+    public static String stringTest2() {
+        // Replace Sachin Tendulkar to Sachin Masterrrr
+        String s = "Sachin Tendulkar".replace("Tendulkar","Masterrrr");
+        return s;
+    }
+    public static String stringTest3() {
+        // Dell to Depp
+        String d = "Dell".replace("ll","pp");
+        return d;
+    }
```

Comment:
Please find my below comments :
- The name for the function is not a meaningful one. 
- Any reason why we we are defining this as a static function?
- This is not an optimised approach. Lets create a generic function irrespective of the string like : 
`String getReplacedString(String originalString, String stringToReplace)`

and then you can have 3 junit5 tests which verifies these 3.